packages/trader/src/Modules/Contract/Containers/contract.tsx

```diff
@@ -1,16 +1,20 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Redirect, withRouter } from 'react-router';
 import { CSSTransition } from 'react-transition-group';
 import { routes } from '@deriv/shared';
 import ErrorComponent from 'App/Components/Elements/Errors';
 import { localize } from '@deriv/translations';
-import ContractReplay from './contract-replay.jsx';
+import ContractReplay from './contract-replay';
 import { observer, useStore } from '@deriv/stores';
 
+type TContract = {
+    match: { isExact: boolean; params: { contract_id: string }; path: string; url: string };
+    history: Array<string>;
```

Comment:
```suggestion
    history: Array<string>;
    history: History;
```

Reply:
could also be this

`history: RouteComponentProps['history'];`

Reply:
Actually they point to the same browser history interface. Up to you what you want to put there

Reply:
I applied your suggestion from above, so there is no need in adding type for history. Thank you)